maddymz/irondb
# US204

Added example route with template and custom JQuery. 

To test:

- check that link in readme directs you to playlist

-  `localhost:3001/example` should direct to page with button that does nothing else you've registered a user, if users registered it displays usernames when clicked.

- `localhost:3001/example/secret`, returns 401 unless logged in.
